src/Apis.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from "react";
+
+const useFlightApi = (url, id = null) => {
```

Comment:
Name should be better : useGetFlightList.

Also, the filename shouldn't end with JSX. 

- JSX if the file returns HTML.
- js/ts if the file returns JS.

Reply:
<img width="988" alt="image" src="https://github.com/piyushukla/Flight-status/assets/9150910/cfedce50-062a-45e3-94ca-2a2594997ded">

font size is terribly disproportionate. Flight number and name should be bigger. 

also, instead of colouring the status text, use background colour. example : https://bootstrapdocs.com/v3.0.3/docs/components/#labels